includes/class-comment.php

```diff
@@ -335,4 +346,29 @@ public static function generate_id( $comment ) {
 			\trailingslashit( site_url() )
 		);
 	}
+
+	/**
+	 * Enqueue scripts for remote comments
+	 */
+	public static function enqueue_scripts() {
```

Comment:
Let's be more selective here: only print these scripts if we're on a single page with comments open.